src/wallet/sqlite.cpp

```diff
@@ -130,6 +131,27 @@ bool SQLiteDatabase::Verify(bilingual_str& error)
         return false;
     }
 
+    // Check our schema version
+    sqlite3_stmt* user_ver_stmt{nullptr};
+    ret = sqlite3_prepare_v2(m_db, "PRAGMA user_version", -1, &user_ver_stmt, nullptr);
+    if (ret != SQLITE_OK) {
+        sqlite3_finalize(user_ver_stmt);
+        error = strprintf(_("SQLiteDatabase: Failed to prepare the statement to fetch sqlite wallet schema version: %s"), sqlite3_errstr(ret));
+        return false;
+    }
+    ret = sqlite3_step(user_ver_stmt);
+    if (ret != SQLITE_ROW) {
+        sqlite3_finalize(user_ver_stmt);
+        error = strprintf(_("SQLiteDatabase: Failed to fetch sqlite wallet schema version: %s"), sqlite3_errstr(ret));
+        return false;
+    }
+    int32_t user_ver = sqlite3_column_int(user_ver_stmt, 0);
+    sqlite3_finalize(user_ver_stmt);
+    if (user_ver != WALLET_SCHEMA_VERSION) {
+        error = strprintf(_("SQLiteDatabase: Unknown sqlite wallet schema version %d. Only version %d is supported"), user_ver, WALLET_SCHEMA_VERSION);
```

Comment:
I changed `WALLET_SCHEMA_VERSION` to `1`, recompiled and was able to open an existing wallet...
Ditto if I create a new wallet with version 10, set it back to 0 and load it.

Reply:
I think this should be fixed now.

Reply:
It is